config.txt

```diff
@@ -93,8 +93,15 @@ null,
  * pool_address	  - Pool address should be in the form "pool.supportxmr.com:3333". Only stratum pools are supported.
  * wallet_address - Your wallet, or pool login.
  * pool_password  - Can be empty in most cases or "x".
+ *
+ * Featured small pools       fee    
+ * pool.minemonero.pro:3333 - 0.9%
+ * pool.usxmrpool.com:3333  - 0.5%
+ * pool.xmr.pt:3333         - 0.4%
+ *
+ * We feature pools up to 1MH/s here. For a more complete list see M5M400's pool list at www.moneropools.com
  */
-"pool_address" : "pool.supportxmr.com:3333",
+"pool_address" : "pool.usxmrpool.com:3333",

```

Comment:
This pool is showing the following message on the start page: 
```
We do not tolerate pool hoppers. If you are identified as one, you will be blocked
 from the pool, and any balance you may accumulate won't be paid. 
Pool hopping isn't fair for anyone, so we can't tolerate it in any form. 
If you aren't sure what pool hopping is, this message is most probably not for you.
```
It is absolutely unclear how this detection is working and I am not sure if some how is mining for a view hours than leave and came back later is identified as pool hopper. IMO a pool should be neutral and not evaluating the miner behave.
@fireice-uk can we remove this pool?